.github/workflows/CI.yml

```diff
@@ -0,0 +1,59 @@
+name: CI
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v*
+  pull_request:
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.3'
+          - '1'
```

Comment:
...so the reason this is failing is that we previously supported 1.3 and 1.4, and now we're supporting 1.3 and 1 (...aka 1.5). There's a dependence on a conveniently chosen seeded rng in the tests that (due to intentional/known) differences between rngs across Julia versions causes a failure in 1.5. 

The correct ultimate fix is to remove that test dependence on the rng, but that fix is orthogonal to the changes introduced by this PR, so I think a good course of action here is to not change the supported Julia versions here (i.e., revert to 1.3 and 1.4) and then file a separate issue for bumping the supported versions to 1.3 and 1 (which would necessarily require fixing that test).

Reply:
...lol, that issue is in fact already filed (as #18).